src/snackbar.css.ts

```diff
@@ -18,6 +18,16 @@ export const snackbar = style([
     },
 ]);
 
+export const snackbarContainer = style([
+    sprinkles({
+        isolation: 'isolate',
+        position: 'relative',
+    }),
+    {
+        zIndex: 1, // Render above everything, including other portals
```

Comment:
Snackbar requires zIndex despite being rendered inside a portal, because we don't want it to be covered by other portals

Reply:
can you explain this case? how can this happen?
In the case of multiple portals, the last opened one will be over the others. Isn't this enough?

Reply:
Open a snackbar, and then open a modal or whatever while the snackbar is still there. You can test it in the private story I've created in this PR.

I mentioned this to design, and they agreed that the snackbar should not be covered by the  new portals